sql-ballerina/tests/client.bal

```diff
@@ -76,26 +80,11 @@ returns Error? = @java:Method {
     'class: "org.ballerinalang.sql.testutils.ClientTestUtils"
 } external;
 
-isolated function nativeQuery(Client sqlClient, string|ParameterizedQuery sqlQuery, typedesc<record {}>? rowType)
-returns stream <record {}, Error> = @java:Method {
-    'class: "org.ballerinalang.sql.testutils.QueryTestUtils"
-} external;
-
-isolated function nativeExecute(Client sqlClient, string|ParameterizedQuery sqlQuery)
-returns ExecutionResult|Error = @java:Method {
-    'class: "org.ballerinalang.sql.testutils.ExecuteTestUtils"
-} external;
-
 isolated function nativeBatchExecute(Client sqlClient, ParameterizedQuery[] sqlQueries)
```

Comment:
Better if we move all external function into the client object

Reply:
`nativeBatchExecute` and `createSqlClient` external functions can not move into the client object because there are some logic perform before calling those external functions.  @niveathika is there any way to resolve this?

Reply:
Ack. Lets keep is out of the client